garden_ai/client.py

```diff
@@ -362,6 +364,126 @@ def register_pipeline(self, pipeline: Pipeline, container_uuid: str) -> str:
         local_data.put_local_pipeline(registered)
         return func_uuid
 
+    def create_minimal_step(
+        self,
+        *,
+        model_name: str,
+        local_path: str,
+        flavor: str,
+        input_type: type,
+        output_type: type,
+        requirements_file: str = "requirements.txt",
+    ) -> Step:
+        with open(requirements_file, "r") as f:
```

Comment:
I'm guessing that for this flow we're more likely to be passing in requirements as a list of strings than as a file